packages/terra-tag/src/Tag.jsx

```diff
@@ -57,7 +64,8 @@ class Tag extends React.Component {
 
   handleKeyUp(event) {
     // Apply focus styles for keyboard navigation
-    if (event.nativeEvent.keyCode === KeyCode.KEY_TAB) {
+    const eventKeyCode = new Set([KeyCode.KEY_TAB, KeyCode.KEY_LEFT, KeyCode.KEY_RIGHT, KeyCode.KEY_RETURN]);
```

Comment:
This code does not seem correct to me.  You are not actually moving the focus so why are we making a state change?

Reply:
changes done. Thanks